notes/Sources/Factory/Factory.swift

```diff
@@ -15,7 +15,7 @@ struct Factory: FactoryProtocol {
   }
   
   func makeNotesListView() -> AnyView {
```

Comment:
Change `AnyView` -> `View` in protocol
and:
```suggestion
  func makeNotesListView() -> some View {
```

Reply:
'some' type cannot be the return type of a protocol requirement

Reply:
but View?